app/scripts/d3-context-menu.js

```diff
@@ -13,8 +13,10 @@ function contextMenu(menu, opts) {
   let initialPos = null;
   let parentStart = null;
 
-  let openCallback,
-    closeCallback;
+  let openCallback;
+
+
```

Comment:
Why two empty lines here?

Reply:
No idea: It's just what `--fix` did. I think I'll grep for sequences of more than 1 newline in a row, and fix them all, but after this.